### ISSUE TITLE
support for databases

### ISSUE THREAD

**Issue by riccardoklinger** [lcalisto/qgis-finddataset-plugin#4]
For the time being the tools offers the search function in folders. It would be great to have a similar functionality for databases like PostGIS

**Comment by lcalisto** [comment]
Yes this could be an interesting enhancement. I don't think it would be problematic to implement since I'm using gdal and ogr. The user should be able to choose between file based datasets and postgis database in the latest the user also needs to select a postgis connection from qgis.
